packages/postgres-database/src/simcore_postgres_database/models/projects_metadata.py

```diff
@@ -47,10 +48,29 @@
         server_default=sa.text("'{}'::jsonb"),
         doc="Reserved for the user to store custom metadata",
     ),
+    sa.Column(
+        "parent_project_uuid",
+        sa.String,
+        nullable=True,
+        doc="If applicable the parent project UUID of this project",
```

Comment:
what does parenthood mean in this context? Can you define that in the doc?

Reply:
ahah do I need to make a drawing? let me know what you mean with this

Reply:
Here I understand that this is the project_uuid of the parent project. Is this correct?

Reply:
I am also not sure I know exactly what it means. I know one project can create another project via the api-server. Is that the only way one project can be the parent of another?

Reply:
right, so a project contains X nodes. one of these nodes uses the API to create jobs, these jobs = projects that are the children of the original project/node that was using the API.